device-connectors/src/testflinger_device_connectors/devices/iotscript/iot-auto-sanity/setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
```

Comment:
This file is necessary due to we need to install this tool on zapper after device-connector copy the folder to zapper

Reply:
I'm not really a Python expert, but would it not be a bit easier to issue a `pip install` on the zapper from a private Git URL instead of copying from the device connector to the Zapper and then doing this?

Reply:
@mz2 I have proposed this method and was rejected by Rex and ...... you XD

Reply:
Leaving a comment here for posterity: we have resolved this confusion I believe successfully last week, and landed on a different design, where Testflinger device connector simply relays on (over RPC) data to the service running on Zapper, which also houses the Zapper deployed part to this.